### PR TITLE
feat(InputNumber): support classNames props

### PR DESCRIPTION
**Pull request by liuyib**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [x] New feature
- [ ] Bug fix
- [x] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [x] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 🔗 Related issue link

- fix: #46841

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |     feat(InputNumber): support classNames props (this props has been supported in 5.7.0, but it cause styles lost and docs not include it, so change to 5.13.0 to support it)     |
| 🇨🇳 Chinese |    feat(InputNumber): 支持 classNames 参数（该参数从 5.7.0 支持，但是会导致样式丢失，并且文档没有透出，因此改为 5.13.0 支持）     |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

**Comment by liuyib** [comment]
根据 @MadCcc 大佬说的：https://github.com/ant-design/ant-design/issues/46841#issuecomment-1884458338, 可以支持 input, prefix, 也可以考虑 handler，handler 需要 rc-input-number 支持，所以需要先给 rc-input-number 发下版：https://github.com/react-component/input-number/pull/617

辛苦大佬们看下~

File: components/input-number/index.en-US.md [hunk 1]
```diff
@@ -42,6 +42,7 @@ Common props ref：[Common props](/docs/react/common-props)
 | addonAfter | The label text displayed after (on the right side of) the input field | ReactNode | - |  |
 | addonBefore | The label text displayed before (on the left side of) the input field | ReactNode | - |  |
 | autoFocus | If get focus when component mounted | boolean | false | - |
+| classNames | Semantic DOM class | Record<[SemanticDOM](#inputnumber), string> | - | 5.13.0 |
```
Comment (MadCcc) [inline components/input-number/index.en-US.md:45]: 不只是 classNames，styles 以及 ConfigProvider 上 InputNumber 配置的 classNames 和 styles 都要支持

File: components/input-number/index.en-US.md [hunk 2]
```diff
@@ -66,6 +67,18 @@ Common props ref：[Common props](/docs/react/common-props)
...
+
+#### InputNumber
+
+| Property       | Description                | Version |
+| -------------- | -------------------------- | ------- |
+| input          | Wrapper of `input` element | 5.13.0  |
+| prefix         | Wrapper of prefix          | 5.13.0  |
+| handlerWrapper | Wrapper of `+-` controls   | 5.13.0  |
+| upHandler      | `+` control                | 5.13.0  |
+| downHandler    | `-` control                | 5.13.0  |
```
Comment (MadCcc) [inline components/input-number/index.en-US.md:80]: handler 拿不准可以先不加，先把 input 和 prefix 做上

Comment (liuyib) [reply components/input-number/index.en-US.md:80]: 好的

File: components/input-number/index.en-US.md (hunk 1 above)
Comment (liuyib) [reply components/input-number/index.en-US.md:45]: 好的

**Comment by afc163** [comment]
Hi @liuyib 

Thank you for your contribution!

We noticed that this PR hasn't been updated in a while. To keep the project active and maintainable, we'll be closing this PR for now.

If you'd like to proceed with these changes, please feel free to open a new PR.

Thanks again for your contribution!

**Pull request closed without merging**